regression/tests/studio/test_studio_grading.py

```diff
@@ -0,0 +1,48 @@
+"""
+Test studio grading
+"""
+from bok_choy.web_app_test import WebAppTest
+from regression.pages.studio.login_studio import StudioLogin
+from regression.pages.studio.grading_studio import GradingPageExtended
+
+from regression.tests.helpers import LoginHelper, get_course_info
+
+
+class StudioGradingTest(WebAppTest):
+    """
+    Test studio grading
+    """
+    def setUp(self):
+        """
+        Initialize the page object
+        """
+        super(StudioGradingTest, self).setUp()
+        self.login_page = StudioLogin(self.browser)
+        LoginHelper.login(self.login_page)
+        self.course_info = get_course_info()
+
+        self.grading_page = GradingPageExtended(
+            self.browser, self.course_info['org'], self.course_info['number'],
+            self.course_info['run'])
+
+        self.grading_page.visit()
+
+    def test_grade_range(self):
+        """
+        Tests default, addition and deletion of grade range
+        """
+        # Default
+        self.assertEquals(
+            self.grading_page.letter_grade('.letter-grade'), 'Pass')
+        # Addition
+        self.grading_page.click_new_grade_button()
+        self.assertEquals(self.grading_page.letter_grade('.letter-grade'), 'A')
+        self.browser.refresh()
```

Comment:
@benpatterson 
Refresh is another case where we verify that the changes are still there even after refreshing/navigating from page. I will add comment so it is clear for readers.